src/cli/commands/beacon.ts

```diff
@@ -36,11 +41,22 @@ export class BeaconNodeCommand implements CliCommand {
           address: options.depositContract
         },
         provider: await this.getProvider(options)
+      },
+      rpc: {
+        apis: this.setupRPC(options.rpc.split(","))
```

Comment:
I think we want a trim here
eg: 
```typescript
this.setupRPC(options.rpc.split(",").map((option) => option.trim()))
```

Reply:
Commander doesn't allow for spaces, it will automatically assume you're issuing another command or flag, so not sure if it is necessary.. Fine to add for saftey though

Reply:
Ok, didn't know that, its fine then.

Reply:
but i think you can wrap in quotes then it allows spaces